src/v/redpanda/admin_server.cc

```diff
@@ -1689,6 +1707,15 @@ admin_server::create_user_handler(std::unique_ptr<ss::httpd::request> req) {
     auto username = security::credential_user(doc["username"].GetString());
     validate_no_control(username(), string_conversion_exception{username()});
 
+    if (is_no_op_user_write(
```

Comment:
I believe we filter off other no-op commands in the raft layer. Any reason why we are doing it in the admin server here?

Reply:
security_frontend doesn't have a `credentials_store`, so isn't in a position to do the check.  One does eventually get a `user_exists` error, but that's happening in the "wait" part of `replicate_and_wait`.

We could do this same check in security_frontend, but it feels kind of equivalent to do it here (it's the one and only place we do user requests)

Reply:
Agreed, thanks for the clarification